test/integration/scheduler/volume_binding_test.go

```diff
@@ -894,12 +886,12 @@ func TestRescheduleProvisioning(t *testing.T) {
 }
 
 func setupCluster(t *testing.T, nsName string, numberOfNodes int, features map[string]bool, resyncPeriod time.Duration, provisionDelaySeconds int, disableEquivalenceCache bool) *testConfig {
-	oldFeatures := make(map[string]bool, len(features))
-	for feature := range features {
-		oldFeatures[feature] = utilfeature.DefaultFeatureGate.Enabled(utilfeature.Feature(feature))
+	featureGatesRestoreFuncs := []func(){}
```

Comment:
Can we remove this and just have every test case set the appropriate feature gates?

Reply:
Hmm, I didn't see any significant difference between the two.
I even feel it better to keep this as is, to avoid setting feature gates everywhere.
Anyway, I don't have strong objections if you'd suggest to remove this.

Reply:
Nm I misread, this fine.